airflow/models/baseoperator.py

```diff
@@ -1706,25 +1701,49 @@ def _validate_kwarg_names_for_mapping(cls: Type[BaseOperator], func_name: str, v
 class MappedOperator(DAGNode):
     """Object representing a mapped operator in a DAG"""
 
-    operator_class: Type[BaseOperator] = attr.ib(repr=lambda c: c.__name__)
+    def __repr__(self) -> str:
+        return (
+            f'MappedOperator(task_type={self.task_type}, '
+            + f'task_id={self.task_id!r}, partial_kwargs={self.partial_kwargs!r}, '
+            + f'mapped_kwargs={self.mapped_kwargs!r}, dag={self.dag})'
```

Comment:
```suggestion
            f'MappedOperator(task_type={self.task_type}, '
            f'task_id={self.task_id!r}, partial_kwargs={self.partial_kwargs!r}, '
            f'mapped_kwargs={self.mapped_kwargs!r}, dag={self.dag})'
```

implicit string concatenation